### ISSUE TITLE
gensim package import issues  

### ISSUE THREAD

**Issue by rsurendraidexcel** [piskvorky/gensim#3544]
Why pakage import issue 

import gensim
![Screenshot from 2024-07-11 00-53-59](https://github.com/piskvorky/gensim/assets/7713535/936e2071-35f7-4cf5-8ec3-730babd888b5)

**Comment by piskvorky** [comment]
@mpenkov what's the plan on that next release? Or else we will keep receiving these tickets over and over :)

**Comment by onedavidwilliams** [comment]
I have the same issue.

**Comment by Crosswind** [comment]
I believe this is a duplicate to #3525 where the same import produced an error. It was then fixed in #3524 and the code was merged ;-) https://github.com/piskvorky/gensim/blob/dc5b5c48e7454fe22cf98ddac60ff85107226f6a/gensim/matutils.py#L23-L26

**Comment by piskvorky** [comment]
Yes, you are right. We've had probably a dozen tickets for this same issue. And a few PRs too. The issue was solved a while ago => merged into `master` => pending a PyPI release.

@mpenkov how about another call to push the release through? Is there anything blocking it?

**Comment by lolpro11** [comment]
For those on Arch, the same issue is occurring with the AUR package. Please push an updated package to the AUR as well.
Thanks.

**Comment by albas99** [comment]
From the discussion above, it seems that this issue has been fixed and is awaiting release from PyPI, I don't know if there's a release window that we'll have to wait for but this is making gensim pretty much unusable with the latest version of scipy.

**Comment by Crosswind** [comment]
@albas99 the release window is specified by the maintainers of this repository. It is up to them to decide when a new version is released and pushed to PyPi. This has nothing to do with PyPi itself.

For my projects, we don't depend on `scipy` directly so we just pinned it to `<1.14.0` to avoid the mentioned problems.

I suggest this issue is closed and we let @piskvorky and @mpenkov do what's necessary to get us a new version which fixes the mentioned problem.

**Comment by mpenkov** [comment]
The blocking issue is failing wheel builds: https://github.com/piskvorky/gensim/actions/workflows/build-wheels.yml

I'm not sure what the problem is there. I've been looking at it occasionally, as time permits, but haven't pinned the cause down yet. Very little things have changed on our end, so it's possible it's a CI issue or another incompatibility between our direct/indirect dependencies.

**Comment by mpenkov** [comment]
@piskvorky We need to get the wheels building again, otherwise we simply cannot release. We could jump on a call, but there isn't much to discuss or plan: once the wheels are building, we can pull the trigger and release relatively quickly.

My intent is to continue chipping away at the problem until I work out what the cause is. It's hard to give an estimate of when that will be, both because we don't yet know what exactly the problem is, and because my personal resources are limited.

To everyone waiting on the release, I'm sorry that it's taking so long. If you're able to shed light on why the wheel builds are failing, then that would be very helpful.

**Comment by Crosswind** [comment]
I have just created a PR with a suggested fix that should make the tests green again. Hope that helps!

**Comment by mpenkov** [comment]
Thank you @Crosswind.

I have been working on the same issue today via #3538 .

There are two problems here:

1. scipy 1.14.0 removed sparsetools, as you have mentioned
2. github CI deprecated the macos-11 runner, so we have to use a newer runner to build the macos wheels

I'm waiting for the CI to complete for #3538 - I expect the above two problems to be fixed now.

**Comment by Crosswind** [comment]
@mpenkov Apologies for missing your latest commits in #3538. Anyways, CI is looking much better now. Fingers crossed this'll go through now.

**Comment by mpenkov** [comment]
Fixed in #3538
